modules/cluster-logging-collector-syslog.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-syslog_{context}"]
+= Sending logs using the Fluentd syslog plug-in (RFC 3164)
+
+You can use the Fluentd *syslog* plug-in to send a copy of your logs to an external syslog server, 
+not the default Elasticsearch. Note the following about the *syslog* plug-in:
+
+* uses syslog protocol (RFC 3164), not RFC 5424;
+* does not support TLS and thus, is not secure;
+* does not provide Kubernetes metadata, systemd data, or other metadata.
+
+[NOTE]
+====
+The Fluentd *syslog* plug-in method is deprecated and will be removed in a future release.
+====
+
+There are two versions of the Fluentd *syslog* plug-in:
+
+* *out_syslog*. The non-buffered implementation, which communicates through UDP, does not buffer data and writes out results immediately. 
+* *out_syslog_buffered*: The buffered implementation, which communicates through TCP, link:https://docs.fluentd.org/buffer[buffers data into chunks]. 
+
+To configure the Fluentd *syslog* plug-in, create a configuration file, called `sysconfig.conf`, with the information needed to forward the logs. Then use that file to create a ConfigMap called `syslog` in the `openshift-logging` Namespace, which {product-title} uses when forwarding the logs. On the receiver, configure the *in_syslog* plug-in to receive the logs from {product-title}. For more information on using the in_forward plug-in, see the link:https://docs.fluentd.org/input/syslog[Fluentd documentation].
+
+[IMPORTANT]
+====
+Changes introduced by the new log forward feature modified the support for Fluentd *syslog* plug-in starting with the {product-title} 4.3 release. In {product-title} 4.3, you create a ConfigMap, as described below, to configure the Fluentd *syslog* plug-in. 
+====
+
+You can use multiple syslog servers by specifying separate `<store>` stanzas in the configuration file.
+
+.Sample `sysconfig.conf`
+[source,yaml]
```

Comment:
Does `[source,yaml]` provide useful highlighting for this?

Reply:
Looks better without the [source,yaml] in asciibinder preview.